mock/chairs/starting_a_group.md

```diff
@@ -0,0 +1,23 @@
+# Starting a Group
+## Menu
+* [Is starting a group the right course of action?]() (below)
+* [Incubation and WICG](#) (below)
+* [Community Groups](#) (below)
+* [Creating a Charter](#) (below)
+* [<- Back to **Chairs**](index.md#)
+
+## Content
+Find out whether starting a new group is the right course of action and how to start a group below!
+
+### Is starting a group the right course of action?
+Add content.
+
+### Incubation and WICG
+Add content.
+
+### Community Groups
+Add content.
+
+### Creating a Charter
```

Comment:
Somewhere we need to capture "Amending a Charter" too.

Reply:
added!